src/PostgREST/DbStructure.hs

```diff
@@ -323,46 +323,55 @@ accessibleTables =
       relname as table_name,
       d.description as table_description,
       (
-        c.relkind IN ('r', 'v','f')
-        AND (pg_relation_is_updatable(c.oid::regclass, FALSE) & 8) = 8
-        OR EXISTS (
-          SELECT 1
-          FROM pg_trigger
-          WHERE
-            pg_trigger.tgrelid = c.oid
-            AND (pg_trigger.tgtype::integer & 69) = 69
+        c.relkind IN ('r','p')
+        OR (
+          c.relkind in ('v','f')
```

Comment:
Did you run this new query on the big schema? Seems the perf won't change, but maybe we should double check to be sure.

Reply:
Just tested it with the big schema and couldn't find relevant differences in the execution time between this branch and main.